daemons/controld/controld_execd.c

```diff
@@ -1802,7 +1802,8 @@ do_lrm_invoke(long long action,
         update_attrd(lrm_state->node_name, CRM_OP_PROBED, XML_BOOLEAN_TRUE,
                      user_name, is_remote_node);
 
-    } else if (pcmk__strcase_any_of(CRM_OP_REPROBE, operation, crm_op, NULL)) {
+    } else if (pcmk__str_eq(crm_op, CRM_OP_REPROBE, pcmk__str_casei)
+               || pcmk__str_eq(operation, CRM_OP_REPROBE, pcmk__str_casei)) {
```

Comment:
I think this will get caught by coccinelle and turned back into `pcmk__strcase_any_of`.  I don't have a way of fixing that.

Reply:
Luckily it doesn't because of a quirk in the cocci file ... it checks for (s1,s2) || (s1,s3), but not for (s2,s1) || (s3, s1). But that's probably a reasonable way of doing it to allow for this sort of thing.